tests/sles4sap/hana_install.pm

```diff
@@ -106,8 +106,8 @@ sub run {
         my @zypper_in = ('install');
         # Check for SAPHanaSR-angi package going to be used
         if (get_var('USE_SAP_HANA_SR_ANGI')) {
-            script_run('[ $(rpm -q SAPHanaSR-doc) ] && rpm -e --nodeps SAPHanaSR-doc');
-            script_run('[ $(rpm -q SAPHanaSR) ] && rpm -e --nodeps SAPHanaSR');
+            script_run('rpm -q SAPHanaSR-doc && rpm -e --nodeps SAPHanaSR-doc');
+            script_run('rpm -q SAPHanaSR && rpm -e --nodeps SAPHanaSR');
```

Comment:
Perhaps it would be a good idea to take this opportunity to replace these 2 `script_run()` for `assert_script_run()`. I think it's mandatory to remove classic packages when configuring angi.